ime/app/src/main/res/values-fr/strings.xml

```diff
@@ -305,6 +305,8 @@
     <string name="key_press_preview_popup_off_summary">Ne pas afficher un aperçu de la touche lors de l\'appui</string>
     <string name="zoom_factor_keys_in_portrait">Ratio de hauteur des touches en mode portrait</string>
     <string name="zoom_factor_keys_in_landscape">Ratio de hauteur des touches en mode paysage</string>
+    <string name="settings_hint_size">Taille des indices</string>
```

Comment:
This will be overwritten by the localization bot. To do translations, you need to use https://crowdin.net .
Just create the default string resource.

Reply:
Okay, will remove it.